src/utils/fetchClient.ts

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Avoid using 'any' type. It's better to replace it with a more specific type or use generics when possible.